bundles/plugins/org.bonitasoft.studio.connectors/src/org/bonitasoft/studio/connectors/ui/property/section/ConnectorSection.java

```diff
@@ -310,6 +313,13 @@ public void handleChange(final ChangeEvent event) {
         updateButtons();
     }
 
+    protected void setTreeFilter(){
```

Comment:
shouldn't this method be abstract ?
This method should return a ViewerFilter instead of delegating the operation of adding the filter in a subclass.